src/topology/continuous_function/t0_sierpinski.lean

```diff
@@ -0,0 +1,77 @@
+/-
+Copyright (c) 2022 Ivan Sadofschi Costa. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Ivan Sadofschi Costa
+-/
+import topology.order
+import topology.continuous_function.basic
+
+/-!
+# Any T0 space embeds in a product of copies of the Sierpinski space
+
+We consider `Prop` with the Sierpinski topology. If `(X, t)` is a topological space, there is a
```

Comment:
I don't think `(X, t)` is appropriate notation, given that it's not what we use in `mathlib`.